### ISSUE TITLE
Tooltip floats despite effect="solid"

### ISSUE THREAD

**Issue by mir3z** [wwayne/react-tooltip#143]
See attached [video](https://github.com/wwayne/react-tooltip/files/371392/tooltip.mp4.zip)

Despite the fact I set `effect="solid"` on tooltip it floats for the first time I hover tooltip trigger

```jsx
<ReactTooltip id={ TOOLTIP_ID } effect="solid" place="right" class="my-tooltip" />
```

**Comment by wwayne** [comment]
Switching from one `place` to other `place` will cause it, you can reproduce it by changing the example react-tooltip `place` from top to right for example.

I don't know how you use the react-tooltip, but I'm thinking optimize this

**Comment by wwayne** [comment]
Fixed in `3.1.0`
